### PR TITLE
edited deploying to cloudflare workers page for wrangler 2.x

### LINKED ISSUE #3188: Update Wrangler section of Deploying to Cloudflare Workers doc
Our CloudFlare Workers doc has a [section on Wrangler](https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-cloudflare-workers#2-set-up-wrangler), but this covers Wrangler version 1. Wrangler 2.0 is now the current version. We should update the doc to support this.

@millsp  identified this docs issue, and can provide the technical information about this.

### PR DESCRIPTION
**Pull request by ObamaTheLlama114**
Updates the [cloudflare workers deploy page](https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-cloudflare-workers) to use wrangler 2.

Fixes #3188

### CODE REVIEW

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 1]
```diff
@@ -73,6 +73,19 @@ Wrangler is the official Cloudflare Worker CLI. You will use it to develop and d
 
 ```terminal
 npx wrangler init
+ ⛅️ wrangler 2.1.15
```
Comment (keerlu) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx]: This looks to be the output for the command? Need to remove this and just explain the input steps, or maybe use the `<CodeWithResult>` component if it's important

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 2]
```diff
@@ -73,6 +73,19 @@ Wrangler is the official Cloudflare Worker CLI. You will use it to develop and d
 
 ```terminal
 npx wrangler init
```
Comment (Jolg42) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx:77]: Maybe mention somewhere explicetely that this uses wrangler v2 and link to their docs?
https://developers.cloudflare.com/workers/wrangler/

**Review by Jolg42 — COMMENTED** [review]
Had a quick look. this looks great, love how it got simpler with wrangler v2

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 3]
```diff
@@ -268,37 +208,38 @@ With Cloudflare Workers, you can't directly access your database because there i
...
 1. In your code editor, add the connection string in the `wrangler.toml` file.
 
-    ```diff file=wrangler.toml
-      name = "prisma-mongodb-cloudflare"
-      type = "webpack"
-      webpack_config = "webpack.config.js"
+   ```diff file=wrangler.toml
+     name = "prisma-mongodb-cloudflare"
+     type = "webpack"
+     webpack_config = "webpack.config.js"
```
Comment (millsp) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx]: Is this still required?

Comment (ObamaTheLlama114) [reply content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx]: The webpack part is not. I will remove it.

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 4]
```diff
@@ -64,117 +64,55 @@ Open your terminal and navigate to a location of your choice. Run the following
...
-npm install -D prisma typescript @cloudflare/wrangler webpack
+npm install -D prisma typescript wrangler
 ```
 
 ## 3. Set up Wrangler
 
-Wrangler is the official Cloudflare Worker CLI. You will use it to develop and deploy to Cloudflare Workers.
+Wrangler is the official Cloudflare Worker CLI. You will use it to develop and deploy to Cloudflare Workers. This guide uses [Wrangler v2](https://developers.cloudflare.com/workers/wrangler/).
+
+First, initialize wrangler. To do this, run the following command in your terminal:
```
Comment (keerlu) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx]:
```suggestion
First, initialize Wrangler. To do this, run the following command in your terminal:
```

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 5]
```diff
@@ -64,117 +64,55 @@ Open your terminal and navigate to a location of your choice. Run the following
...
-And then install `@cloudflare/workers-types` with npm:
+We also want to use TypeScript, so answer yes.
 
 ```terminal
-npm install -D @cloudflare/workers-types ts-loader@8.3.0
+Would you like to create a Worker at src/index.ts?
 ```
 
-<Admonition type="info">
+We want to use the `Fetch Handler`.
```
Comment (keerlu) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx]:
```suggestion
Select the `Fetch Handler` option.
```

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 6]
```diff
@@ -64,117 +64,55 @@ Open your terminal and navigate to a location of your choice. Run the following
...
+This will ask you a few questions.
 
 ```terminal
-npx wrangler login
-Allow Wrangler to open a page in your browser? [y/n] y
+Would you like to use git to manage this Worker? (y/n)
 ```
 
-You can now verify that you're logged in by running `npx wrangler whoami`.
+We want to use git, so answer yes.
```
Comment (keerlu) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx]: (we seem to capitalize elsewhere, so I'd suggest this even though it's lower case in the command output)

```suggestion
We want to use Git, so answer yes.
```

**Review by keerlu — APPROVED** [review]
LGTM now from a writing point of view - just some small comments

**Review by millsp — APPROVED** [review]

**Pull request merged**